utils/moreutils/Makefile

```diff
@@ -0,0 +1,61 @@
+#
+# Copyright (C) 2016 Nikil Mehta <nikil.mehta@gmail.com>
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=moreutils
+PKG_VERSION:=0.59
+PKG_RELEASE:=1
+
+PKG_SOURCE:=$(PKG_NAME)_$(PKG_VERSION).orig.tar.gz
+PKG_SOURCE_URL:=http://http.debian.net/debian/pool/main/m/moreutils/
+PKG_MD5SUM:=a48e11c3197bf79a7bfaa121423e64626e3381d9fedc91d606e9724ae498d1b4
+
+PKG_MAINTAINER:=Nikil Mehta <nikil.mehta@gmail.com>
+PKG_LICENSE:=GPL-2.0
+PKG_LICENSE_FILES:=COPYING
+
+PKG_BUILD_DIR:=$(BUILD_DIR)/$(PKG_NAME)-$(PKG_VERSION).orig
+
+include $(INCLUDE_DIR)/package.mk
+
+define Package/moreutils
+  SECTION:=utils
+  CATEGORY:=Utilities
+  TITLE:=additional Unix utilities
+  URL:=https://joeyh.name/code/moreutils/
+  DEPENDS:=+perl +perlbase-file +perlbase-getopt +perlbase-io +perlbase-ipc +perlbase-posix
+endef
+
+define Package/moreutils/description
+	This is a growing collection of the Unix tools that nobody thought
+	to write long ago, when Unix was young.
+	.
+	So far, it includes the following utilities:
+	- chronic: runs a command quietly unless it fails
+	- combine: combine the lines in two files using boolean operations
+	- errno: look up errno names and descriptions
+	- ifdata: get network interface info without parsing ifconfig output
+	- ifne: run a program if the standard input is not empty
+	- isutf8: check if a file or standard input is utf-8
+	- lckdo: execute a program with a lock held
+	- mispipe: pipe two commands, returning the exit status of the first
+	- parallel: run multiple jobs at once
+	- pee: tee standard input to pipes
+	- sponge: soak up standard input and write to a file
+	- ts: timestamp standard input
+	- vidir: edit a directory in your text editor
+	- vipe: insert a text editor into a pipe
+	- zrun: automatically uncompress arguments to command
+endef
+
+define Package/moreutils/install
+	$(INSTALL_DIR) $(1)/usr/bin
+	$(INSTALL_BIN) $(PKG_BUILD_DIR)/{chronic,combine,errno,ifdata,ifne,isutf8,lckdo,mispipe,parallel,pee,sponge,ts,vidir,vipe,zrun} $(1)/usr/bin/
```

Comment:
PKG_INSTALL forces "make install" to be executed, and the "install" rule tries to install manpages.  As I mention below, I can't get the manpages to compile, so install will fail.

I think a better solution is to use PKG_INSTALL:=1 and to update my patch to also remove the installation of manpages.  I went ahead and did that - let me know what you think.